static/styles/settings.css

```diff
@@ -1192,19 +1192,15 @@ input[type=checkbox].inline-block {
     transform: translateY(-50%);
 }
 
-#id_realm_create_stream_permission {
+#id_realm_create_stream_permission,
+#id_realm_bots_allowed_for_non_admins {
     width: 100%;
 }
 
 .m-t-10 {
     margin-top: 10px;
 }
 
-.create-stream-dropdown {
-    margin-right: 10px;
-    margin-bottom: 5px;
-}
-
```

Comment:
Why is this CSS removed?

Reply:
OK; I'd probably put that in a commit after the rest with that explanation in the commit message.  

As a general rule, it's great to notice things like that, but it's a lot faster for code review if each commit just does one thing well, and not other things you noticed along the way.